include/config.h

```diff
@@ -285,6 +280,11 @@
 
 #define SYNC_INTERVAL 8
 
+/* Sync time (minimum time between syncing in ms, time is halfed for -M main
+   nodes): */
+
+#define SYNC_TIME 18000000LLU                      /* 18000000 = 30 minutes */
```

Comment:
`30 * 60 * 1000` could be easier to understand

Reply:
true